src/sql/workbench/parts/notebook/notebook.css

```diff
@@ -42,6 +42,7 @@
 
 .notebookEditor .labelOnLeftContainer {
 	min-width: 100px;
+	max-width: 300px;
```

Comment:
Minor: I'd love if it was 250px, but frankly this works for now and we will solve properly with the toolbar refactor which will let us have a separate Database dropdown.